pypeit/core/wavecal/templates.py

```diff
@@ -154,15 +159,22 @@ def pypeit_arcspec(in_file, slit):
     return wv_vac, np.array(iwv_calib['spec'])
 
 
-def pypeit_identify_record(iwv_calib, binspec, specname, gratname, dispangl):
+def pypeit_identify_record(iwv_calib, binspec, specname, gratname, dispangl, outdir=None):
     """From within PypeIt, generate a template file if the user manually identifies an arc spectrum
 
     Args:
-        iwv_calib (dict): Wavelength calibration returned by final_fit
-        binspec (int): Spectral binning
-        specname (str): Name of instrument
-        gratname (str): Name of grating
-        dispangl (str): Dispersion angle
+        iwv_calib : dict
```

Comment:
This is numpy doc syntax, so you need to change "Args" to "Parameters" and remove the indentation.  I.e.,

```
Parameters
---------

iwv_calib : dict
    Wavelength calibration returned by final_fit
...
```

Reply:
See... failures are inevitable... I've fixed this one...